public/assets/main.css

```diff
@@ -241,6 +253,27 @@ table.center {
   margin: 0.2rem auto;
 }
 
+#chatlog {
+  height: 200px;
+}
+@media only screen and (min-width: 900px) {
+  #homepage #chatlog {
+    height: 350px;
+  }
+}
+
+.game_row > .card_header {
```

Comment:
is this necessary to have in css?

Reply:
Not for this to work here. It would be useful if it’s needed in other places.